packages/ibm-products/src/components/Carousel/Carousel.tsx

```diff
@@ -352,6 +354,18 @@ const Carousel = React.forwardRef<HTMLDivElement, CarouselProps>(
       }
     }, [rightFadedEdgeRef, rightFadedEdgeColor]);
 
+    useIsomorphicEffect(() => {
+      if (scrollRef?.current) {
+        setIsScrollableElement(
+          scrollRef.current.scrollHeight > scrollRef.current.clientHeight
+        );
+      }
+    }, []);
+
+    const scrollableElementAttributes = isScrollableElement
+      ? { 'aria-label': 'Carousel element', tabIndex: 0, role: 'region' }
```

Comment:
There should be a way to pass this `aria-label` in to the component so it is not hard coded. I'll open a follow up issue for this.